modules/doc/content/help/faq/apptainer.md

```diff
@@ -0,0 +1,160 @@
+# Civet-like Environment
```

Comment:
"CIVET" instead? If so, this will need to be corrected several times. Noting once here.

Reply:
Actually I was thinking of changing this to Apptainer Environment.